packages/arb-token-bridge-ui/src/util/TokenListUtils.ts

```diff
@@ -157,12 +158,19 @@ export async function fetchTokenListFromURL(tokenListURL: string): Promise<{
       }
     })
 
-    if (!validateTokenList(data)) {
+    const isValid = validateTokenList(data)
```

Comment:
Can we maybe validate them on line 113. I don't think we need invalid list anywhere, no?

Reply:
the list can only be validated after data is fetched